scripts/pscis_tidy.R

```diff
@@ -41,7 +41,10 @@ utm <- 10
 #   ) %>%
 #   sf::st_write('data/inputs_extracted/mergin_backups/form_pscis_v70.gpkg', append=FALSE)
 
-# read in cleaned amalgamated pscis form
+
+#---------------------pscis clean and QA only--------------------------
+
+# read in amalgamated pscis form
 form_pscis <- sf::st_read(dsn= paste0('../../gis/', dir_project, '/data_field/2023/form_pscis_2023.gpkg')) %>%
   st_transform(crs = 26900 + utm) %>%
   poisspatial::ps_sfc_to_coords(X = 'easting', Y = 'northing') %>%
```

Comment:
`poisspatial::ps_sfc_to_coords `changes from sf object to a dbl_df - not usually helpful.  there were issues areound this in `form_fiss_site_tidy.R` so it was changed to more explicit calls for coords. might not be a problem in this script but good to be aware

https://gist.github.com/NewGraphEnvironment/9e20702275c63bf6f15304944a85d535